README.md

```diff
@@ -386,6 +386,16 @@ checkImplicitDependencies {
 }
 ```
 
+These tasks are created by default, but are not configured to run as part of `check`.  To do that, add to the root `/build.gradle`:
+
+```gradle
+allprojects {
+    apply plugin: 'java-library'
```

Comment:
Without this line, my internal test repo fails with:

```
1: Task failed with an exception.
-----------
* Where:
Build file '/Volumes/git/my-project/build.gradle' line: 77

* What went wrong:
A problem occurred evaluating root project 'my-project'.
> Could not get unknown property 'checkUnusedDependencies' for root project 'my-project' of type org.gradle.api.Project.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
```

Maybe there's an alternate pattern that we should recommend instead, of only applying to subprojects that are Java type?

Reply:
I probably wouldn't be prescriptive in this documentation about putting this in `allprojects` - that's something that consumers can decide for themselves.

I'd just write something like:

```diff
+ tasks.check.dependsOn tasks.checkUnusedDependencies
```